### ISSUE TITLE
Collector service left some Alammex/Compx limit orders even if they have been cancelled/filled (was: Remove Alammex limit orders from maximum value engine)

### ISSUE THREAD

**Issue by ipaleka** [asastats/channel#1051]
By Grinnmarr in [Discord](https://discord.com/channels/906917846754418770/910524302531649566/1268556576814272562):

>  Why am I seeing Alammex as an option? I think some code got pushed on accident or something. I wish I could take advantage of some of those trades listed but alas Alammex is no more.

<img src="https://github.com/user-attachments/assets/78279ee8-de9f-4693-a35f-b506624dbb2d" width=500>

-----
ipaleka — Today at 3:33 PM
> The thing is that those limit orders are onchain (of we have a bug), so basically we just need to change the link... 

Grinnmarr — Today at 3:36 PM
> Well, I check multiple times a day and it's the first time I'm seeing them so ... Idk.

ipaleka — Today at 3:37 PM
> That's a valuable information then, thank you! As I said, we updated limit orders today so that act probably revealed the problem.

-----

**Comment by moloch10** [comment]
Not sure if this is related but I'll drop an observation I just made. I'm holding a token and the swap recommendation is Alammex. I've never seen this exchange so I clicked the link and got the closure notice and [new location](https://app.compx.io/swap) as well of course. Are those limit orders affecting the suggestion for this swap; in which case this is the same thing? 😅

**Comment by ipaleka** [comment]
> Not sure if this is related but I'll drop an observation I just made. I'm holding a token and the swap recommendation is Alammex. I've never seen this exchange so I clicked the link and got the closure notice and [new location](https://app.compx.io/swap) as well of course. Are those limit orders affecting the suggestion for this swap; in which case this is the same thing? 😅

This original report is divided into two issues, here's the one you're referring to: https://github.com/asastats/channel/issues/1052

Basically, we have got a bug (this issue) that is sort of difficult to track. After the dex data re-fetching from the scratch the issue is resolved, so it seems that the problem is in Coillector's service during real-time update, but I simulated the exact block from the atomic transaction which affected this Grinnmarr's problem and the dex entry was correctly removed afterwards. 

I'm about to close this issue as the data is correctly presented both in the Grinnmarr case and for our testing accounts - the newly initiated changes in our testing account are correctly recorded.

The [second issue](https://github.com/asastats/channel/issues/1052) is resolved, the old Alammex text and links are simply renamed to Compx as both monitor the same dApp, if you place a limit order on Alammex website -which is still available - it will end up in Compx UI too.
